### PR TITLE
doc(readme): Update main readme with latest LTS pointers

### PR DESCRIPTION
**Pull request by timtay-microsoft**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -172,8 +172,8 @@ Below is a table showing the mapping of the LTS branches to the packages release
 
 | Maven Package | Github Branch | LTS Status | LTS Start Date | Maintenance End Date | Removed Date |
 | :-----------: | :-----------: | :--------: | :------------: | :------------------: | :----------: |
-| [2020-01-27](https://github.com/Azure/azure-iot-sdk-java/releases/tag/LTS_01_2020) | lts_01_2020   | Active     | 2020-01-27     | 2020-06-30            | 2020-12-31   |
-| [2019-01-24](https://github.com/Azure/azure-iot-sdk-java/releases/tag/LTS_01_2019_Rev01) | LTS_01_2019_Rev01 | Deprecated | 2019-01-24 | 2019-06-30 | 2019-12-31 |
+| [2020-7-7](https://github.com/Azure/azure-iot-sdk-java/releases/tag/lts_7_2020) | lts_07_2020   | Active     | 2020-7-7     | 2020-12-31            | 2021-6-30   |
+| [2020-01-27](https://github.com/Azure/azure-iot-sdk-java/releases/tag/LTS_01_2020) | lts_01_2020   | Deprecated     | 2020-01-27     | 2020-06-30            | 2020-12-31   |
```
Comment (prmathur-microsoft) [inline readme.md:176]: I don't think end date aligns with what is broadly communicated. It needs to be atleast one year as I recall. @sharmasejal can you help?

Comment (timtay-microsoft) [reply readme.md:176]: We can check with her, sure. My understanding is that we officially support each LTS for 6 months, then we leave it deprecated for 6 months

Comment (sharmasejal) [reply readme.md:176]: maintenance end date in a year from start date
and removed date in 3 years from start date

**Pull request merged**